### DISCUSSION TITLE
Add Vulkan Renderer for phones with Snapdragon SoCs

### DISCUSSION THREAD

**Discussion by PayToUse** [ppy/osu#26982] [Ideas]
## Description
Since osu! lazer has only one renderer for android phones, I always thinked about having a vulkan renderer the game for better performance.

## Why Vulkan?
Since most Snapdragon processors use a Vulkan graphics driver, it also gives exclusive performance features to various apps. I've been also sharing this just now because I've been experiencing lag spikes during gameplay and on the menu since a couple of updates.

![Screenshot_20240203_071512_sh ppy osulazer](https://github.com/ppy/osu/assets/67496494/2ce5addb-b128-4b0a-a654-7e1b0dba995a)

**Comment by Flamiii** [comment]
Vulkan is planned in the future (see https://github.com/ppy/osu/discussions/26383#discussioncomment-8010278).

**Comment by bdach** [comment]
https://github.com/ppy/osu-framework/pull/6105 needs to happen first so that we can proceed with this.

**Reply by Aurora-Nasa-1** [reply]
> [ppy/osu-framework#6105](https://github.com/ppy/osu-framework/pull/6105) needs to happen first so that we can proceed with this.

now？
